docs/edgeql/delete.rst

```diff
@@ -13,7 +13,7 @@ The ``delete`` command is used to delete objects from the database.
 Clauses
 -------
 
-Deletion statements support ``filter``, ``order by``, and ``offset/limit``
+Deletion statements support ``filter``, ``order by``, ``offset``, and ``limit``
```

Comment:
When I first saw this as `offset/limit`, I thought that it implied that those clauses couldn’t be used separately. But since they can, I thought it would be better to list `offset` and `limit` as separate items.

Reply:
Good point.